BD-4852-2 WhatsApp Product Messages (#9896)
braze-inc/braze-docs: _docs/_user_guide/message_building_by_channel/whatsapp/whatsapp_campaign/product_messages.md

{% endalert %}

When you send a WhatsApp product message to a user, the user goes on the following customer journey:

1. The user receives your product or catalog message in WhatsApp.
2. The user adds products to their cart directly from WhatsApp.
3. The user taps **Place order** in WhatsApp.
4. Your website or app recieves the cart data from Braze and generates a checkout link.
5. The user is directed to your website or app to complete their checkout.

When users add items to their cart through catalog messages, Braze receives webhook data for follow-up actions.

## Requirements

| Requirement | Description |
| --- | --- |
| WhatsApp Business Account | To use WhatsApp product messages, you must have a WhatsApp Business Account connected with Braze. |
| Meta catalog | You need to set up a Meta catalog in your Commerce Manager. |
| Term compliance | Comply with the [Meta Commerce Terms and Policies](https://www.facebook.com/policies_center/commerce). |
{: .reset-td-br-1 .reset-td-br-2 role="presentation" }

## Product message templates

{% tabs %}
{% tab Catalog messages %}

> Display your entire product catalog in an interactive format.

Reviewer: For ease of translation, I would keep these as complete thoughts!

```suggestion
Catalog messages display your entire product catalog in an interactive format.
```

{% alert note %}
You don't need to make additional product selections in Braze, as the catalog connection is managed by Meta and thus is inherited into your product catalog.
{% endalert %}

{% endtab %}
{% tab Multi-product messages %}

> Highlight specific products from your catalog, with up to 30 highlighted items per message. Currently, there isn't an integrated product selector, so you must manually reference your Meta catalog to obtain product SKUs.

Reviewer: ```suggestion
Multi-product messages highlight specific products from your catalog, with up to 30 highlighted items per message. Currently, there isn't an integrated product selector, so you must manually reference your Meta catalog to obtain product SKUs.
```

{% alert important %}

> Multi-product message templates currently have a header display bug. Meta is working on a fix.

Reviewer: I would suggest more plain language here (instead of "bug" -> "known issue")

```suggestion
There’s a known header display issue with multi-product message templates on Meta. Meta is aware of the issue and working on a fix.
```

2. Follow Meta's instructions to [connect your Meta catalog](https://www.facebook.com/business/help/1953352334878186?id=2042840805783715) to your Braze-connected WhatsApp Business Account by assigning the "Manage Catalog" permission in Meta Business Manager. 

![Meta "Catalogs" page with an arrow pointing at the "Assign partner" button for the catalog called "sweeney_catalog".]({% image_buster /assets/img/whatsapp/meta_catalog.png %}){: style="max-width:80%;"}

Make sure to use the Braze Business Manager ID, `332231937299182`, as the partner business ID.

![Window to share a catalog with a partner that contains fields to enter a partner business ID and assign the permission "Manage catalog".]({% image_buster /assets/img/whatsapp/share_meta_catalog.png %}){: style="max-width:60%;"}

{: start="3"}

> 3. Select your Meta catalog settings. You must select “Show catalog icon in chat header” to send catalog messages.

Reviewer: ```suggestion
3. Select your Meta catalog settings. You must select **Show catalog icon in chat header** to send catalog messages.
```
